__tests__/app.test.js

```diff
@@ -403,3 +403,27 @@ describe("GET /api/articles (sorting queries)", () => {
       });
   });
 });
+
+describe("GET /api/articles (topic query)", () => {
+  test.only("200: Respond with object that sorted_by created_at by default in desc order,and topic =cat", () => {
```

Comment:
You need to remove .only 


There is another happy path test - there is a topic that exists that has no articles attached to it. Should return an empty array and a 200!